### DISCUSSION TITLE
How to use this?

### DISCUSSION THREAD

**Discussion by sprnza** [polarmutex/beancount-language-server#374] [Q&A]
Sorry for maybe too general question but still
I've installed everything needed and LspInfo shows "1 client attached to this buffer" but I can't figure out how to get syntax highlighted and autocompletion working. When I type "Expenses" I do have autocomplition popup appearing but once I add ":" symbol the popup dissappears.
It's seems like I'm missing somehting obvious here, but I'm new to neovim so could you please point me where to look for such kind of questions.

**Comment by sprnza** [comment]
Okay, it kinda started working after I installed "vim-beancount" plugin but autocompletion still stops working after : symbol for some reason

**Comment by avanc** [comment]
Hello,
I got also stuck while trying to use this LSP in Neovim.
The [documentation](https://github.com/polarmutex/beancount-language-server?tab=readme-ov-file#neovim) mentions the dotfiles by @polarmutex . Although the link is broken, I found this one: [beancount.lua](https://github.com/polarmutex/dotfiles/blob/master/dot_config/nvim/lua/polarmutex/lsp/servers/beancount.lua)
But that file uses some node.js implementation of the server.

Can someone point me to a working example?

Thanks in advance
Sven
